### ISSUE TITLE
Enemies get their weapons back when loading saves

### ISSUE THREAD

**Issue by solek1312** [shinyquagsire23/OpenJKDF2#59]
When you use force pull on enemies they lose their guns.
But if you save and then load the save, they get them back, allowing you to get infinite ammo

**Comment by solek1312** [comment]
I am being told in the original version they also get their guns back and this can also be abused for infinite ammo.
However in the original JK they continue to behave as if they don't have any guns and never shoot

**Comment by shinyquagsire23** [comment]
Huh, yeah if the original game has the infinite ammo bug then I might hold off on that particular fix (or make an ifdef for gameplay-altering bugfixes), but if the AI is getting reset when the original game didn't then that one sounds like a regression

**Comment by solek1312** [comment]
Yeah, the bug has to do with AI becoming aggressive again, not guns.
Having guns not reappear at  would've been a nice optional fix for the clearly incorrect original behaviour

**Comment by StrikerMan780** [comment]
Yeah, I think this was something that was fixed in a patch? (As in, not JK.EXE 1.0)

**Comment by Vortikai** [comment]
Tested OpenMOTS level 1 with the stormtroopers outside the base, force pulling all their weapons, quick saving, aborting the mission, and reloading the save. Unable to reproduce, probably fixed now.

[v0.8.7](https://github.com/shinyquagsire23/OpenJKDF2/releases/tag/v0.8.7)
